hack/common.sh

```diff
@@ -663,7 +663,7 @@ function os::build::os_version_vars() {
       # Try to match the "git describe" output to a regex to try to extract
       # the "major" and "minor" versions and whether this is the exact tagged
       # version or whether the tree is between two tagged versions.
-      if [[ "${OS_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)(\.[0-9]+)?([-].*)?$ ]]; then
+      if [[ "${OS_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)(\.[0-9]+)?([-,+].*)?$ ]]; then
```

Comment:
@smarterclayton my fault I didn't want to commit and I forgot to revert this change.
In my clone `$OS_GIT_VERSION` is `v1.0.6+f1a2c89-dirty` and it doesn't match the regex.
I didn't have time to  debug  so I modified the regex... The intent was not to commit.